addons/medical_treatment/functions/fnc_treatment.sqf

```diff
@@ -30,18 +30,20 @@ if !(_this call FUNC(canTreat)) exitWith {false};
 private _config = configFile >> QGVAR(actions) >> _classname;
 
 // Get treatment time from config, exit if treatment time is zero
-private _treatmentTime = if (isText (_config >> "treatmentTime")) then {
-    GET_FUNCTION(_treatmentTime,_config >> "treatmentTime");
+private _treatmentTimeConfig = ["treatmentTime", "treatmentTimeTrained"] select ([_medic] call FUNC(isMedic));
```

Comment:
```suggestion
private _treatmentTimeConfig = ["treatmentTime", "treatmentTimeTrained"] select (([_medic] call FUNC(isMedic)) && {!isNull (_config >> "treatmentTimeTrained")});
```
I think we need this as well to ensure compatiblity with other mods adding medical actions that may not have this config. Otherwise it will just get a 0 and exit.